src/stores/webSocket.js

```diff
@@ -52,11 +71,16 @@ export const useWebSocket = defineStore('webSocket', () => {
       return
     }
     const parsedData = camelcaseKeysDeep(JSON.parse(data))
-    await processSocketMessage(parsedData)
+    if (parsedData.subscription === 'Object' && parsedData.payload?.hash === subscribedTransactionId.value) {
```

Comment:
Could this be explained via extra const binding?

Reply:
I tried my best but it was really hard to come up with the name for the first part half of that if statement